include/tst_test.h

```diff
@@ -23,6 +23,9 @@
 #endif /* __TEST_H__ */
 
 #include <unistd.h>
+#ifndef __GLIBC__
+#include <limits.h>
+#endif
 
```

Comment:
There are seven cases failed due to PATH_MAX undeclared:
access04.c getcwd04.c getcwd02.c listxattr02.c llistxattr02.c
madvise09.c umount02.c
So I think add "#include <limits.h>" to tst_test.h more better.